docs/setup/independent/install-kubeadm.md

```diff
@@ -51,6 +51,20 @@ The pod network plugin you use (see below) may also require certain ports to be
 open. Since this differs with each pod network plugin, please see the
 documentation for the plugins about what port(s) those need.
 
+## Installing ebtables ethtool
+
+if you see error warning while run ```kubeadm init```
+
+```
+[preflight] WARNING: ebtables not found in system path                          
+[preflight] WARNING: ethtool not found in system path                           
+
+```
+You may missing these tools on your linux , please try to installed them .
```

Comment:
1) may -> may be
2) "linux ," extra space 
3) installed -> install
4) them . extra space